pkg/kubelet/kubelet.go

```diff
@@ -2964,7 +2979,7 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod, podStatus *kubecontaine
 		}
 		if kl.isPodResizeInProgress(allocatedPod, podStatus) {
 			// If a resize is in progress, make sure the cache has the correct state in case the Kubelet restarted.
```

Comment:
I think this comment is no longer relevant since you're backfilling the conditions in HandlePodAdditions, right?

Reply:
yeah I think the part about if the kubelet restarted is not relevant anymore, took it out